tests/conftest.py

```diff
@@ -0,0 +1,21 @@
+"""Pytest fixtures for the Tsunami agent"""
+import pathlib
+
+import pytest
+from ostorlab.agent import definitions as agent_definitions
+from ostorlab.runtimes import definitions as runtime_definitions
+
+from agent import tsunami_agent
+
+@pytest.fixture(scope='function', name='tsunami_agent')
+def fixture_tsunami_agent(mocker):
```

Comment:
Great that you refactored this.
This can be replaced with the simple agent_mock.